### PR TITLE
Fix run script for meshctl installation to get latest

### PR DESCRIPTION
**Pull request by Sodman**

### CODE REVIEW

**Review by ashleywang1 — APPROVED** [review]

File: hack/install.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ else
...
+  GLOO_MESH_VERSIONS=$(curl -s https://storage.googleapis.com/storage/v1/b/meshctl/o?pageToken=CiB2Mi4xLjAtYmV0YTIvbWVzaGN0bC1saW51eC1hbWQ2NA | any_python -c "import sys; from distutils.version import StrictVersion, LooseVersion; from json import loads as l; obj = l(sys.stdin.read()); items = obj['items'];  all_releases = [release['name'].split('/')[0] for release in items]; releases = []; [releases.append(n) for n in all_releases if n not in releases]; filtered_releases = list(filter(lambda release_string: len(release_string) > 0 and StrictVersion.version_re.match(release_string[1:]) != None, releases)); filtered_releases.sort(key=LooseVersion, reverse=True); print('\n'.join(filtered_releases))")
```
Comment (fernandosanchezs) [inline hack/install.sh:17]: Is this fetching a specific page (hence the constant pageToken) to get this version?

Comment (nrjpoddar) [reply hack/install.sh:17]: How stable is the pagination token?

Comment (Sodman) [reply hack/install.sh:17]: @fernandosanchezs - This is currently page 2, yes.

@nrjpoddar I can't find any documentation that clarifies either way. In hindsight perhaps a better route would be to do something like `?maxResults=2000` (default 1000) What do you think?

Comment (nrjpoddar) [reply hack/install.sh:17]: yes
No magic tokens ever in a script..

**Pull request merged**